msal/build.gradle

```diff
@@ -124,6 +134,38 @@ task sourcesJar(type: Jar) {
     destinationDir = reporting.file("$project.buildDir/outputs/jar/")
 }
 
+task createMsalCoverageReportTask() {
```

Comment:
This probably doesn't need to be a task of its own. I don't think we ever need to call this directly, right? 

It can just be a method that's called on the build variants, and just go directly in the `android` section of the file -- IIRC we are already looping over build variants somewhere in this file so you can just follow that pattern.

Reply:
generation of all the new tasks are now implemented in already loop-over sub section of android block of build.gradle file